src/main/java/com/google/sps/servlets/AnalyzedThread.java

```diff
@@ -0,0 +1,96 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.Random;
+
+public class AnalyzedThread {
+  public static String[] titlesArray = {"Where is this supposed to go?",
```

Comment:
nit: prefer working with `ImmutableList` (from Guava) as they're usually clearer to use than Arrays, and are immutable.

```java
public static final ImmutableList<String> titlesArray = ImmutableList.of("string1", "String2", ...);
```